lib/src/signed_video_h26x_common.c

```diff
@@ -843,16 +843,69 @@ update_gop_hash(void *crypto_handle, gop_info_t *gop_info)
 
   return status;
 }
+
+svrc_t
+initialize_and_update_gop_level_hash(signed_video_t *self, uint8_t *hash_list, int idx)
```

Comment:
Wrapping `openssl_init_hash(...)` and `openssl_update_hash(...)` in a function like this adds no value. Remove.